v3/docs/res/community.md

```diff
@@ -7,9 +7,12 @@ how we can improve HSE, or contribute to development.
 ## Report a Bug
 
 You can file an [issue](https://github.com/hse-project/hse/issues)
-to report an bug.  In doing so, please provide all the information
+to report a bug in the HSE library.  In doing so, please provide all the information
```

Comment:
This change seems to discount that this would also be where you file issues with tests, the CLI, tools, etc.

Reply:
I suppose to be as accurate as possible I can remove the link altogether and just state that an issue should be filed in the repo containing the code that is the source of the bug.

Reply:
Makes sense to me